timewarp_gl/plugin.cpp

```diff
@@ -677,28 +724,37 @@ class timewarp_gl : public threadloop {
 
 #ifndef NDEBUG
 		if (log_count > LOG_PERIOD) {
-			printf("\033[1;36m[TIMEWARP]\033[0m Swap time: %5fms\n", (float)(afterSwap - beforeSwap) * 1000);
-			printf("\033[1;36m[TIMEWARP]\033[0m Motion-to-display latency: %3f ms\n", float(imu_to_display.count()) / 1e6);
-			printf("\033[1;36m[TIMEWARP]\033[0m Prediction-to-display latency: %3f ms\n", float(predict_to_display.count()) / 1e6);
-			printf("\033[1;36m[TIMEWARP]\033[0m Render-to-display latency: %3f ms\n", float(render_to_display.count()) / 1e6);
-			std::cout<< "Timewarp estimating: " << std::chrono::duration_cast<std::chrono::milliseconds>(GetNextSwapTimeEstimate() - lastSwapTime).count() << "ms in the future" << std::endl;
+            const double time_swap = std::chrono::duration_cast<std::chrono::milliseconds>(time_after_swap - time_before_swap).count();
+            const double latency_mtd = imu_to_display.count()/1e6;
+            const double latency_ptd = predict_to_display.count()/1e6;
+            const double latency_rtd = render_to_display.count()/1e6;
+            const time_type time_next_swap = GetNextSwapTimeEstimate();
+            const double timewarp_estimate = std::chrono::duration_cast<std::chrono::milliseconds>(time_next_swap - time_last_swap).count();
+
+            std::cout << "\033[1;36m[TIMEWARP]\033[0m Swap time: " << time_swap << "ms" << std::endl
+			          << "\033[1;36m[TIMEWARP]\033[0m Motion-to-display latency: " << latency_mtd << "ms" << std::endl
+			          << "\033[1;36m[TIMEWARP]\033[0m Prediction-to-display latency: " << latency_ptd << "ms" << std::endl
+		              << "\033[1;36m[TIMEWARP]\033[0m Render-to-display latency: " << latency_rtd << "ms" << std::endl
+			          << "Timewarp estimating: " << timewarp_estimate << "ms in the future" << std::endl;
```

Comment:
This wording has always confused me. Can you make it something like "Next swap in: xyz ms"?